Manifest.toml

```diff
@@ -1,5 +1,3 @@
-# This file is machine-generated - editing it directly is not advised
```

Comment:
Guess this is not Julia 1.3? Would be good to make sure especially with the whole JLL situation.

Reply:
I mean I am pro just deleteing all Manifests, but which Manifest would you like?

Reply:
I think what you have is fine. We test on 1.3 so whatever that generates is good.